ui/lib/objects.ts

```diff
@@ -82,6 +83,11 @@ export class FluxObject {
     }
   }
 
+  // TODO: this should be merged with kind
```

Comment:
Could you please clarify what this comment means?

Reply:
Good shout. Done.